hack/local-up-operator.sh

```diff
@@ -167,7 +167,7 @@ if ! $KUBECTL_BIN get ns "$NAMESPACE" &>/dev/null; then
 fi
 
 echo "info: installing crds"
-$KUBECTL_BIN create -f manifests/crd.yaml
+$KUBECTL_BIN replace -f manifests/crd.yaml
```

Comment:
`apply`?

Reply:
the crd.yaml is too big to be saved in annotation(apply will do this)